sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -11,7 +11,7 @@
 ### Other Changes
 
 * Update OpenTelemetry dependencies
-  ([#34128](https://github.com/Azure/azure-sdk-for-net/pull/34128)
+  ([#34128](https://github.com/Azure/azure-sdk-for-net/pull/34128))
```

Comment:
@TimothyMothra Would you also scan through the file and fix other places?

e.g. 
![image](https://user-images.githubusercontent.com/17327289/220478610-0c3deb49-dcad-4ce0-84ec-f6938385a38f.png)

![image](https://user-images.githubusercontent.com/17327289/220478651-f440a91a-bed2-4268-8fc3-bed6f7c27e86.png)